pallets/loans/src/lib.rs

```diff
@@ -1302,26 +1224,35 @@ impl<T: Config> Pallet<T> {
         Ok(total_borrow_value)
     }
 
-    fn collateral_asset_value(
+    fn current_collateral_balance(
         borrower: &T::AccountId,
         asset_id: AssetIdOf<T>,
-    ) -> Result<FixedU128, DispatchError> {
+    ) -> Result<BalanceOf<T>, DispatchError> {
         if !AccountDeposits::<T>::contains_key(asset_id, borrower) {
-            return Ok(FixedU128::zero());
+            return Ok(BalanceOf::<T>::zero());
         }
         let deposits = Self::account_deposits(asset_id, borrower);
         if !deposits.is_collateral {
-            return Ok(FixedU128::zero());
+            return Ok(BalanceOf::<T>::zero());
         }
         if deposits.voucher_balance.is_zero() {
-            return Ok(FixedU128::zero());
+            return Ok(BalanceOf::<T>::zero());
         }
         let exchange_rate = Self::exchange_rate_stored(asset_id)?;
         let underlying_amount =
             Self::calc_underlying_amount(deposits.voucher_balance, exchange_rate)?;
         let market = Self::market(asset_id)?;
         let effects_amount = market.collateral_factor.mul_ceil(underlying_amount);
 
+        Ok(BalanceOf::<T>::saturated_from(effects_amount))
+    }
+
+    fn collateral_asset_value(
+        borrower: &T::AccountId,
+        asset_id: AssetIdOf<T>,
+    ) -> Result<FixedU128, DispatchError> {
```

Comment:
shall we use BalanceOf<T> to keep consistency

Reply:
all funcs in loan suffix with `value` including `borrow|collateral|liquidation_value` are price in usdt as `FixedU128`, and funcs with `*_balance` are amount